components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java

```diff
@@ -1458,6 +1460,14 @@ private OpenAPI injectOtherScopesToDefaultScheme(OpenAPI openAPI) throws APIMana
             if (defaultScheme == null) {
                 SecurityScheme newDefault = new SecurityScheme();
                 newDefault.setType(SecurityScheme.Type.OAUTH2);
+                OAuthFlows newDefaultFlows = new OAuthFlows();
```

Comment:
Shall we add a comment on what we are doing here. ie: Populating the default security scheme.

Reply:
fixed with 0d3ff9c